tests/backup/backup_test.go

```diff
@@ -573,21 +560,20 @@ var _ = Describe("{BasicBackupCreation}", func() {
 		opts[SkipClusterScopedObjects] = true
 		log.Info(" Deleting deployed applications")
 		ValidateAndDestroy(contexts, opts)
-		backupDriver := Inst().Backup
-		for _, namespace := range bkpNamespaces {
-			backupName = fmt.Sprintf("%s-%s", BackupNamePrefix, namespace)
-			backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
-			log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
-			DeleteBackup(backupName, backupUID, orgID, ctx)
-		}
-		for _, namespace := range bkpNamespaces {
-			restoreName = fmt.Sprintf("%s-%s", "test-restore", namespace)
-			ctx, err := backup.GetAdminCtxFromSecret()
-			log.FailOnError(err, "Fetching px-central-admin ctx")
-			err = DeleteRestore(restoreName, orgID, ctx)
-			dash.VerifyFatal(err, nil, fmt.Sprintf("Deleting Restore %s", restoreName))
-		}
+
 		DeleteCloudAccounts(backupLocationMap, cloudCredName, cloudCredUID, ctx)
+
+		log.Infof("Deleting registered clusters for admin context")
+		DeleteCluster(SourceClusterName, orgID, ctx)
+		DeleteCluster(destinationClusterName, orgID, ctx)
```

Comment:
Here DeleteCloudAccounts will delete src and dest cluster, backup loc and cloud credential.
We do not need to call DeleteCluster again.

Reply:
Done